README.md

```diff
@@ -0,0 +1,14 @@
+# QA-system for music-related questions
+Hi! This system has been developed as a project for the RUG course Language Technology (2019).
+The following people have been part of this development:
+- Nathan van Beelen (s...)
+- Kevin Kelly (s...)
+- Galina Lesnic (s...)
+- Tom Veldhuis (s3451771)
+## Requirements for running this system:
+- Python3 (and pip3)
+- SpaCy (can be installed by running `pip3 install --user spacy`)
```

Comment:
How should we go about this btw? Because I suppose you want to set `en_core_web_md` back to `en`. In which case the README should probably be updated as well. (So might as well do that now)

Reply:
For now, that does not matter since `en_core_web_md` works in either the case that it has been to referred to as `en` or not.